src/core/resource.ts

```diff
@@ -223,7 +223,7 @@ export function createResource<S>(config: DataTemplate<S> = createMemoryTemplate
 	}
 
 	function getOrRead(options: ResourceOptions): S[] | undefined {
-		const { pageNumber, query, pageSize } = options;
+		const { pageNumber = 1, query, pageSize } = options;
```

Comment:
Defaults the pageNumber to `1` (which it probably always should be if there is not one set)

Reply:
👍